From c63189188ed3b600d8839edb74cc8aa15c252ee9 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 23 Feb 2015 22:18:27 +0000
Subject: [PATCH 4/5] lang: Add to make (^Makefile, ^Makefile.Debug,
 ^Makefile.Release)

All additions are full filename patterns. The .Debug and .Release ones
are used by Qt.
---
 src/lang.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lang.c b/src/lang.c
index 1c78d7c..c1d3152 100644
--- a/src/lang.c
+++ b/src/lang.c
@@ -39,7 +39,7 @@ lang_spec_t langs[] = {
     { "lisp", { "lisp", "lsp" } },
     { "lua", { "lua" } },
     { "m4", { "m4" } },
-    { "make", { "Makefiles", "mk", "mak" } },
+    { "make", { "^Makefile", "^Makefile.Debug", "^Makefile.Release", "Makefiles", "mk", "mak" } },
     { "markdown", { "markdown", "mdown", "mdwn", "mkdn", "mkd", "md" } },
     { "mason", { "mas", "mhtml", "mpl", "mtxt" } },
     { "matlab", { "m" } },
-- 
2.3.0

